.github/workflows/require-allow-edits.yml

```diff
@@ -0,0 +1,14 @@
+name: Require “Allow Edits”
+
+on: [pull_request]
+
+jobs:
+  _:
+    name: "Require “Allow Edits”"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: ljharb/require-allow-edits@main
```

Comment:
Is it typical to use a branch name instead of a SHA? Doesn't this allow you to arbitrarily change the functionality without a PR here? I don't know if I like that.

Reply:
It's very typical, yes; it does allow that (it's how bugfixes and improvements can be landed automatically); it's totally fine to use a sha if you're concerned about it.

Reply:
This is a bit weird given what we have said elsewhere. Since it is a private repository rather than a TC39 repository, can we use a SHA?

Reply:
@codehag it's a repository owned by a current editor, and this repo is owned by the editors. When the action isn't owned by a current editor, we'd immediately update this to use the SHA instead of the branch.